src/core/nginx.go

```diff
@@ -116,6 +119,11 @@ func (n *NginxBinaryType) UpdateNginxDetailsFromProcesses(nginxProcesses []*Proc
 	defer n.workersMapMutex.Unlock()
 	n.nginxWorkersMap = map[string][]*proto.NginxDetails{}
 
+	n.statusUrlMutex.Lock()
+	defer n.workersMapMutex.Unlock()
```

Comment:
this is already called on line 119 `defer n.workersMapMutex.Unlock()`

Reply:
yeah I have the local change but didn't push